### PR TITLE
fix(#450): hx-popover open on render

### PR DESCRIPTION
**Pull request by 100stacks**
* Fixes Issue #450 
* `<hx-popover>` open on render did not align popover properly.

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Building with commit 2f332ca3eb7ca99f76f60259b53ab835ac9096c2

https://app.netlify.com/sites/helix-ui/deploys/5cccce408ffa5b000825c6cb

File: docs/components/popover/test.html [hunk 1]
```diff
@@ -368,7 +400,36 @@ <h2>Within Scrollable Container</h2>
     </hx-div>
   </section>
 
-  <h2>Deprecated Markup</h2>
+  <section>
+      <header>
```
Comment (cathysiller) [inline docs/components/popover/test.html]: fix indentation pls, i think its two spaces over too much starting at `<header>`

File: docs/components/popover/test.html [hunk 2]
```diff
@@ -8,7 +8,39 @@
...
-    <h2>Appearance</h2>
+    <header>
+      <h2 id="button-popover-open-page-load">Button with Popover Open on Page Load</h2>
+    </header>
+    <hx-div class="hxLg">
+      <button class="hxBtn" id="openButtonPopover">
+        Toggle
+      </button>
+      <hx-popover
+        id="demoOpenPopoverTest"
```
Comment (cathysiller) [inline docs/components/popover/test.html]: not necessary, but can alphabetize block

File: docs/components/popover/test.html [hunk 3]
```diff
@@ -368,7 +400,36 @@ <h2>Within Scrollable Container</h2>
...
+                  </p>
+                </hx-div>
+              </hx-popover>
+            </th>
+          </tr>
+        </thead>
+      </table>
+    </section>
+
+    <h2>Deprecated Markup</h2>
```
Comment (cathysiller) [inline docs/components/popover/test.html]: fix indentation here as well

File: docs/components/popover/test.html (hunk 1 above)
Comment (100stacks) [reply docs/components/popover/test.html]: ok, indentation fixed

File: docs/components/popover/test.html (hunk 2 above)
Comment (100stacks) [reply docs/components/popover/test.html]: block alphabetized

**Pull request merged**